### PR TITLE
upgpkg(main): clvk, emscripten

### PR DESCRIPTION
**Pull request by truboxl**

### CODE REVIEW

**Comment by 2096779623** [comment]
Why not use `TERMUX_PKG_AUTO_UPDATE`?

**Comment by truboxl** [comment]
Patches break quite often and some packages need to manually taken care of. Next time to implement custom auto update.

**Review by thunder-coding — APPROVED** [review]

**Comment by thunder-coding** [comment]
> Those 4 build jobs are run in parallel, likely on different computers, so that shouldn't be an issue.

Well I was talking about the auto-update workflow which test builds the package for all 4 architectures before pushing them to the master branch. This is run sequentially.

**Comment by buttaface** [comment]
> I was talking about the auto-update workflow which test builds the package for all 4 architectures before pushing them to the master branch. This is run sequentially.

It looks like [the auto-updater only test builds for aarch64](https://github.com/termux/termux-packages/runs/7130972288?check_suite_focus=true#step:3:2133)? Anyway, I was thinking of the final build once this repo is updated, those are run in parallel.

File: packages/clvk/build.sh [hunk 1]
```diff
@@ -2,16 +2,15 @@ TERMUX_PKG_HOMEPAGE=https://github.com/kpet/clvk
...
+_COMMIT=d416170a674622f20bac04b7dd33b68bfb4096f6
 TERMUX_PKG_SRCURL=https://github.com/kpet/clvk.git
 TERMUX_PKG_GIT_BRANCH=main
 TERMUX_PKG_BUILD_DEPENDS="vulkan-loader-android, vulkan-headers"
 TERMUX_PKG_DEPENDS="libc++"
 TERMUX_PKG_SUGGESTS="ocl-icd"
 TERMUX_PKG_HOSTBUILD=true
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
--DCMAKE_BUILD_TYPE=MinSizeRel
 -DLLVM_TABLEGEN=$TERMUX_PKG_HOSTBUILD_DIR/bin/llvm-tblgen
```
[the remark this answers is not in the document]
Comment (truboxl) [reply packages/clvk/build.sh:14]: I initially put this to see how much it saved the size. Not much has saved. Performance also dropped. I think shipping a "driver" should prioritise performance. It also should match upstream build config (RelWithDebInfo but that can reach over 100MB in package size, longer build time and space waste at CI side, so the second choice is Release).

[the remark this answers is not in the document]
Comment (truboxl) [reply packages/clvk/build.sh:14]: Example of using RelWithDebInfo: https://github.com/truboxl/termux-packages/actions/runs/2366856551

**Review by MrAdityaAlok — APPROVED** [review]

**Pull request merged**